metadata-ingestion/src/datahub/ingestion/extractor/schema_util.py

```diff
@@ -317,17 +336,41 @@ def emit(self) -> Generator[SchemaField, None, None]:
                 merged_props.update(self._schema.other_props)
                 merged_props.update(schema.other_props)
 
-                tags = None
+                # Parse meta_mapping
+                meta_aspects: Dict[str, Any] = {}
+                if self._meta_mapping_processor:
+                    meta_aspects = self._meta_mapping_processor.process(merged_props)
+
+                tags = []
+                if self._schema_tags_field:
+                    for tag in merged_props.get(self._schema_tags_field, []):
+                        tags.append(self._tag_prefix + tag)
+
+                meta_tags_aspect = meta_aspects.get(Constants.ADD_TAG_OPERATION)
+                if meta_tags_aspect:
+                    tags += [
+                        tag_association.tag[len("urn:li:tag:") :]
+                        for tag_association in meta_tags_aspect.tags
+                    ]
+
                 if "deprecated" in merged_props:
                     description = (
                         f"<span style=\"color:red\">DEPRECATED: {merged_props['deprecated']}</span>\n"
                         + description
                         if description
                         else ""
                     )
-                    tags = GlobalTagsClass(
-                        tags=[TagAssociationClass(tag="urn:li:tag:Deprecated")]
-                    )
+                    tags += [
```

Comment:
this doesn't seem right. tags is supposedly list of strings ?

Reply:
Done.